builtin/providers/google/resource_compute_disk.go

```diff
@@ -34,16 +34,21 @@ func resourceComputeDisk() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"project": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+
 			"size": &schema.Schema{
 				Type:     schema.TypeInt,
 				Optional: true,
 				ForceNew: true,
 			},
 
-			"type": &schema.Schema{
+			"self_link": &schema.Schema{
```

Comment:
Kinda. I did a massive pass-through of the docs and found a few places where we had missed arguments or attribute outputs, so now they are ordered equivalent to the docs.